test/connection/connection_with_grpc_test.cpp

```diff
@@ -35,44 +35,81 @@ using type_signatures::Asset;
 using type_signatures::SimpleAsset;
 using type_signatures::Peer;
 
+class connection_with_grpc_test : public testing::Test {
+protected:
 
-TEST(ConnectionWithGrpc, Transaction_Add_Domain){
-    logger::setLogLevel(logger::LogLevel::Debug);
-
-    connection::initialize_peer();
-
-    auto server = []() {
+    void serverVerifyReceive() {
         connection::iroha::Sumeragi::Verify::receive([](const std::string &from, ConsensusEvent &event) {
             std::cout << event.transaction().DebugString() << std::endl;
-            ASSERT_STREQ( event.transaction().senderpubkey().c_str(),              "karin");
-            ASSERT_STREQ( event.transaction().domain().name().c_str(),              "name");
-            ASSERT_STREQ( event.transaction().domain().ownerpublickey().c_str(), "pubkey1");
+            ASSERT_STREQ(event.transaction().senderpubkey().c_str(), "karin");
```

Comment:
May you also extract such strings as a const variables, so it's easier to find/use/fix